File: Day-20/41A.Translation.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    string s, t;
    cin >> s >> t;
    string reversed_s = s;
    reverse(reversed_s.begin(), reversed_s.end());
    cout << (reversed_s == t ? "YES" : "NO") << endl;
    return 0;
}

File: Day-20/41A.Translation.java
import java.util.Scanner;

public class Translation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.next();
        String t = scanner.next();
        String reversed_s = new StringBuilder(s).reverse().toString();
        System.out.println(reversed_s.equals(t) ? "YES" : "NO");
    }
}
